app/common/src/services/__test__/Backend.test.ts

```diff
@@ -1,11 +1,11 @@
-import { Rfc3339DateTime } from 'enso-common/src/utilities/data/dateTime'
 import { describe, expect, it, test } from 'vitest'
+import { Rfc3339DateTime } from '../../utilities/data/dateTime.js'
```

Comment:
Why did we start needing `.js` in imports?

Reply:
More importantly, why do we have `.js` files instead of `.ts`

Reply:
> Why did we start needing `.js` in imports?

we don't, but it is (was?) good practice, since allowing `.js` to be ommitted is actually a feature specific to node.js (and node.js compatible runtimes)

> More importantly, why do we have `.js` files instead of `.ts`

this is a quirk of typescript - it typically accepts `.js` as extensions, not `.ts`, as it does not rewrite import URLs (or at least did not until recently) - and the compiled output is typically expected to have `.js` extension.

Reply:
(let me know if i should remove the `.js` extensions anyway since most/all bundlers don't require the extension)

Reply:
I prefer without extensions